ExcelExtensions/Models/Columns/InAndOutColumn.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Dominic Schira <domshyra@gmail.com>. All Rights Reserved.
+
+using ExcelExtensions.Models.Columns.Import;
+using System.Collections.Generic;
+using static ExcelExtensions.Enums.Enums;
+
+namespace ExcelExtensions.Models.Columns
+{
+    /// <summary>
+    /// Pre defined export location based on colom order, finds colums based on names provided
+    /// </summary>
+    public class InAndOutColumn : UninformedImportColumn
```

Comment:
I still hate this name.... MirrorColumn? ImportAndExportColumn? This part still needs some work...